k8s-deployment.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: schedulertest
+  labels:
+      app: schedulertest 
+      phase: test
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: schedulertest
+  template:
+    metadata:
+      labels:
+        app: schedulertest
+        tier: backend
+        phase: test
+    spec:
+      containers:
+      - name: scheduler-container
+        image: boomzero/night-watch:scheduler
```

Sao lai co boomzero o day?

---

```suggestion
        image: boomzero/night-watch:scheduler
```
cái này là t up lên dockerhub của t, 
team có dockerhub chung k nhỉ